.gitmodules

```diff
@@ -839,3 +839,6 @@
 [submodule "libraries/helpers/ticks"]
 	path = libraries/helpers/ticks
 	url = https://github.com/adafruit/Adafruit_CircuitPython_Ticks
+[submodule "libraries/drivers/scd4x"]
+	path = libraries/drivers/scd4x
+	url = https://github.com/adafruit/Adafruit_CircuitPython_SCD4X.git
```

Comment:
Not sure about the `.git` but it seems to work fine.

Reply:
Actually it's the Ticks library that's missing the `.git`. But it does work either way.